### ISSUE TITLE
"Do you want to save metadata changes?"

### ISSUE THREAD

**Issue by azazellochg** [I2PC/scipion#1859]
**Describe the bug**
I don't understand why this message always pops up when I display classes in scipion viewer and then close the window. What changes? What metadata? I just opened a set of classes. Can we get rid of it?

**Comment by delarosatrevin** [comment]
Yes, we should! It came from the past, when the showj was exclusively an Xmipp tool and when you modify a metadata file, it ask if you can to save the changes. Sorting by a column is considered a change, that's why it ask...but we should remove that behavior.
